cmd/server.go

```diff
@@ -47,6 +47,9 @@ func newServeCmd(stderr io.Writer) *cobra.Command {
 	serveCmd := &cobra.Command{
 		Use:   "server",
 		Short: "Run FeatureBase.",
+		FParseErrWhitelist: cobra.FParseErrWhitelist{
```

Comment:
We can probably remove this now that we handle deprecation....

Reply:
Indeedly do, removing